app/controllers/message_drafts_controller.rb

```diff
@@ -32,9 +32,12 @@ def destroy
     # redirect_path = @message.original_message.present? ? message_thread_path(@message.original_message.thread) : message_drafts_path
     redirect_path = @message.original_message.present? ? message_thread_path(@message.original_message.thread) : message_threads_path
 
-    @message.destroy
-
-    redirect_to redirect_path, notice: "Správa bola zahodená"
+    if @message.not_yet_submitted?
+      @message.destroy
+      redirect_to redirect_path, notice: "Správa bola zahodená"
+    else
+      redirect_to redirect_path, alert: "Správu nie je možné zmazať po odoslaní"
```

Comment:
@luciajanikova je to ze po odoslani alebo pocas odosielania alebo ako to je?

Reply:
Po zaradeni na odoslanie (tak to volame v aplikacii) - t.j. ani pocas odosielania, ani po odoslani.